LICENSE.md

```diff
@@ -67,58 +67,59 @@ The User may not limit access to the Mod in any way, except as provided by the G
 The User may decompile compiled binaries of the Mod.  
 The User may examine the Mod's source code.
 
-5. Right to distribute
+5. No right to distribute
 --------
 
-The Author reserves the right to distribute the Mod.  
-The Author may designate a person or organization as a Distributor.  
-A Distributor is granted the right to distribute the Mod.  
-The Author may require that a Distributor provides a direct link to the Codebase.  
-The User may not distribute the Mod without being granted distribution rights from the Author.  
-The Author may revoke distribution rights from any Distributor.  
+The Author reserves the right to transmit verbatim copies of ("Distribute") the Mod.  
+Rights to distribute the Mod must be granted in a separate license which must be explicitly obtained from the Author.
```

Comment:
No. This makes it too difficult for the author to allow someone to distribute. It would exclude things like distributing your mod via CurseForge (because technically CurseForge is distributing the Mod via their servers...but the Author has designated CurseForge as a their chosen distribution method, so it works the way I worded it.

The way I wrote this, getting permission can be almost any form. It could even be a verbal contract (difficult to prove if it came to court, but verbal contracts are a thing).